probes/pom.xml

```diff
@@ -26,10 +29,21 @@
             <version>${hdr-histogram.version}</version>
         </dependency>
         <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.10</version>
+        </dependency>
+
+        <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>${junit.version}</version>
-            <scope>test</scope>
+            <exclusions>
```

Comment:
These exclusions fix a dependency problem between Hamcrest, JUnit and Mockito which occurs if you run all tests from all modules in your IDE.